CreatubblesAPIClientIntegrationTests/Spec/Group/EditGroupResponseHandlerSpec.swift

```diff
@@ -61,35 +61,35 @@ class EditGroupResponseHandlerSpec: QuickSpec
                     }
                 }
             }
-            
-            it("Should edit group when name and avatar_id are passed")
-            {
-                guard let groupIdentifier = TestConfiguration.testGroupIdentifier,
-                      let creationIdentifier = TestConfiguration.testCreationIdentifier
-                else { return }
-                
-                let data = EditGroupData()
-                data.name =  TestConfiguration.testEditGroupName
-                data.avatarCreationIdentifier = creationIdentifier
-                
-                let sender = TestComponentsFactory.requestSender
-                waitUntil(timeout: 10)
-                {
-                    done in
-                    _ = sender.login(TestConfiguration.username, password: TestConfiguration.password)
-                    {
-                        (error: Error?) -> Void in
-                        expect(error).to(beNil())
-                        _ = sender.send(EditGroupRequest(identifier: groupIdentifier, data:data), withResponseHandler: EditGroupResponseHandler()
-                        {
-                            (error) -> (Void) in
-                            expect(error).to(beNil())
-                            sender.logout()
-                            done()
-                        })
-                    }
-                }
-            }
+            //TODO: uncomment/remove when status is determined on api - for now creating with creation's id as avatar does not work
```

Comment:
there should be a way with quick to simply disable a test, instead of commenting it out. 

I think it's just adding an 'x' before the "it". 
So instead of commenting you can just do: 
```
          xit("Should edit group when name and avatar_id are passed")
``` 
and that would disable the test when being run. Not sure if it's on the "it" or the "describe", think it can be both, might refer to Quick docs though.